frappe/integrations/doctype/webhook/webhook.json

```diff
@@ -1,13 +1,12 @@
 {
  "actions": [],
- "autoname": "naming_series:",
+ "autoname": "format:HOOK-{####}",
```

Comment:
`format:` and naming_series are not equivalent. 

```suggestion
 "autoname": "HOOK-{####}",
```


Either ways this in weird. 

Webhooks should have better descriptive names, so "prompt" makes the most sense.